app/notifications/receive_notifications.py

```diff
@@ -30,28 +32,59 @@ def receive_mmg_sms():
 
     inbound_number = strip_leading_forty_four(post_data['Number'])
 
-    potential_services = dao_fetch_services_by_sms_sender(inbound_number)
-
-    if len(potential_services) != 1:
-        current_app.logger.error('Inbound number "{}" from MMG not associated with exactly one service'.format(
-            post_data['Number']
-        ))
-        statsd_client.incr('inbound.mmg.failed')
+    potential_services = fetch_potential_services(inbound_number, 'mmg')
+    if not potential_services:
         # since this is an issue with our service <-> number mapping, we should still tell MMG that we received
-        # succesfully
+        # successfully
         return 'RECEIVED', 200
 
     statsd_client.incr('inbound.mmg.successful')
 
     service = potential_services[0]
 
-    inbound = create_inbound_mmg_sms_object(service, post_data)
+    inbound = create_inbound_sms_object(service,
+                                        content=format_mmg_message(post_data["Message"]),
+                                        from_number=post_data['MSISDN'],
+                                        provider_ref=post_data["ID"],
+                                        date_received=post_data.get('DateRecieved'),
+                                        provider_name="mmg")
 
+    tasks.send_inbound_sms_to_service.apply_async([str(inbound.id), str(service.id)], queue=QueueNames.NOTIFY)
     current_app.logger.info('{} received inbound SMS with reference {}'.format(service.id, inbound.provider_reference))
```

Comment:
i know you didn't write the firetext one originally but can we duplicate this logging line to the `receive_firetext_sms` function?